applications/ParticleMechanicsApplication/custom_conditions/particle_based_conditions/mpm_particle_base_load_condition.cpp

```diff
@@ -32,20 +32,15 @@ namespace Kratos
      * Shape function values in given point. This method calculate the shape function
      * vector in given point.
      *
-     * @param rPoint point which shape function values have to
-     * be calculated in it.
-     *
-     * @return Vector of double which is shape function vector \f$ N \f$ in given point.
-     *
     */
-    Vector& MPMParticleBaseLoadCondition::MPMShapeFunctionPointValues( Vector& rResult, const array_1d<double,3>& rPoint )
+    void MPMParticleBaseLoadCondition::MPMShapeFunctionPointValues( Vector& rResult) const
```

Comment:
I would probably rename this functions in sense of accessing and modifying shape functions